### PR TITLE
EZP-29681: Clicking Confirmed items closes UDW

### PR DESCRIPTION
**Pull request by dew326**
**Jira ticket:** https://jira.ez.no/browse/EZP-29681

### CODE REVIEW

**Comment by lserwatka** [comment]
@dew326 good one! Finally 👍

**Review by lserwatka — APPROVED** [review]

File: src/modules/universal-discovery/components/selected-content/selected.content.component.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default class SelectedContentComponent extends PureComponent {
      * @memberof SelectedContentComponent
      */
     togglePopup() {
-        this.setState((state) => ({ isPopupVisible: !state.isPopupVisible && !!state.items.length }));
+        this.setState((state) => ({ isPopupVisible: !state.isPopupVisible && !!this.props.items.length }));
```
Comment (tischsoic) [inline src/modules/universal-discovery/components/selected-content/selected.content.component.js]: I'm not sure if it matters here, but maybe:
```js
this.setState((state, props) => ({ isPopupVisible: !state.isPopupVisible && !!props.items.length }));
```

Comment (sunpietro) [reply src/modules/universal-discovery/components/selected-content/selected.content.component.js]: Nice catch!

**Review by sunpietro — APPROVED** [review]

File: src/modules/universal-discovery/components/selected-content/selected.content.component.js (hunk 1 above)
Comment (dew326) [reply src/modules/universal-discovery/components/selected-content/selected.content.component.js]: In this case, it doesn't but fixed.

**Review by micszo — APPROVED** [review]

**Pull request merged**